packages/typo3-docs-theme/assets/sass/_text.scss

```diff
@@ -1,7 +1,25 @@
-
+a {
+    color:inherit;
+    text-underline-offset: 25%;
+    &:hover, &:focus {
+        text-decoration-thickness: 2px;
+    }
+}
+*:focus-visible {
+    @include focus();
```

Comment:
What problem is solved with this?

Reply:
If you currently use tab on the documentation page you cannot see where the focus is. Try using Tab on the new rendering, the focus is clear and the keyboard can be used for navigation. This is a big step toward A11y